src/components/missions-page/MissionScrollBar.scss

```diff
@@ -0,0 +1,55 @@
+@import '../../style/constants';
+
+.MissionScrollBar{
+    .card{
+        display: flex;
+        background-color: #2c3e50;
+        min-width: 100%;
+        min-height: auto;
+        overflow-x: auto;
+        padding: 10px 0;
+        img {
+            display: none;
+        }
+        @media (min-width: 768px) {
+            overflow-x: hidden;
+            flex-wrap: wrap;
+            justify-content: center;
+            img {
+                display: inline;
+                height: 200px;
+                width: 200px;
+            }
+        }
+    }
+
+    .card--content {
+        background-color: $button-primary;
+        min-width: auto;
+        margin: 0px 5px 5px 5px;
```

Comment:
This `5px` spacing seems to be a thing that we repeat everywhere. Should it be a constant?